### DISCUSSION TITLE
Rocky-8.5 doesn't boot

### DISCUSSION THREAD

**Discussion by Sysadmin-CAOS** [cobbler/cobbler#3022] [Q&A]
Hello,

My Cobbler system runs an older version 2.8.0-4 (ufff) and now I'm trying to add a new distro Rocky-8.5. After import DVD, I have create a new "distro" and I have configure "kernel" and "initrd" files. Also, I have configured "Kickstart Metadata" parameter to link to correct point.
However, when I boot my node, process hangs because Cobbler is trying to find "images/install.img" and "LiveOS/squashfs.img". First file exists (???), but second not, because Rocky-8.5 DVD version doesn't contain that tree structure... 

So, how can I deploy my node with Rocky-8.5? (and, now, I can't upgrade my Cobbler server because I have configured all nodes and, also, there is a Puppet server in the same server.

Thanks a lot!!

**Comment by SchoolGuy** [comment]
Okay first things first... Do you have a backup? And if yes can you restore it? Because Cobbler is not able to be downgraded and there are no plans to support this atm. If you are running it in a VM maybe take a Snapshot?

Regarding your Rocky 8.5 problem: Well according to our signature file (`distro_signatures.json`) and the code both files should be present on a Rocky Linux ISO. If this is not the case anymore a PR with adjusted signatures would be highly appreciated.

Regarding the Upgrade to a newer Cobbler version... I would highly recommend creating a small lab environment which is identical from a settings point and then try to play with this a bit. Maybe use the Terraform provider or Ansible to set it up in an automated way so you can throw away the env and quickly rebuild it? I am sadly not aware of a way to automate the Cobbler Setup with Puppet.
